### PR TITLE
Builder add redists and ignore dlls so they are not shipped but installed.

### PR DESCRIPTION
**Pull request by lfdversluis**

### CODE REVIEW

**Comment by lfdversluis** [comment]
retest this please

**Comment by lfdversluis** [comment]
@devos50 Please take a look at this PR and verify all is ok.

**Comment by lfdversluis** [comment]
`test_download_torrent_from_url (Tribler.Test.API.test_download.TestDownload) ... `

**Comment by lfdversluis** [comment]
retest this please

**Comment by lfdversluis** [comment]
retest this please

**Comment by lfdversluis** [comment]
Build on both builders and installed both on proxmox 144, seems to work fine.

**Comment by devos50** [comment]
Looks good, I have no comments on this PR.

**Comment by devos50** [comment]
It's a bit annoying that we are shipping three runtimes but we might reduce this number if we are going to recompile libraries.

**Comment by lfdversluis** [comment]
@devos50 True

**Comment by whirm** [comment]
@lfdversluis could you create an issue about reducing the number of VC redist dependency? this is getting a bit out of hand :D

File: Tribler/Main/Build/Win/tribler.nsi [hunk 1]
```diff
@@ -177,10 +177,14 @@ Section "!Main EXE" SecMain
...
+ ; Install MSVCR 2008, 2010, 2012
  SetOutPath "$INSTDIR"
- File vc_redist.exe
- ExecWait "$INSTDIR\vc_redist.exe /q /norestart"
+ File vc_redist_90.exe
+ ExecWait "$INSTDIR\vc_redist_90.exe /q /norestart"
+ File vc_redist_100.exe
+ ExecWait "$INSTDIR\vc_redist_100.exe /q /norestart"
+ File vc_redist_110.exe
+ ExecWait "$INSTDIR\vc_redist_110.exe /q /norestart"
```
Comment (whirm) [inline Tribler/Main/Build/Win/tribler.nsi]: Could you annotate what depends on each of them so we don't have to redo your work when fixing this?

File: Tribler/Main/Build/Win/setuptribler.py [hunk 2]
```diff
@@ -67,8 +67,10 @@
                         "optimize": 2,
                         "skip_archive": True,
                         "dist_dir": os.path.join("dist", "installdir"),
-                        "dll_excludes": ["mswsock.dll", "MSVCR90.dll", "msvcr71.dll",
-                                         "IPHLPAPI.dll", "NSI.dll", "WINNSI.dll", "WTSAPI32.dll"]
+                        "dll_excludes": ["mswsock.dll", "MSVCR90.dll", "MSVCP90.dll",
+			"msvcr71.dll", "msvcp71.dll", "MSVCR100.dll", "MSVCP100.dll",
```
Comment (whirm) [inline Tribler/Main/Build/Win/setuptribler.py]: This doesn't look like the right indentation for the nested list.

**Comment by lfdversluis** [comment]
retest this please

**Comment by lfdversluis** [comment]
`FAIL: test_i2i_running (Tribler.Test.test_i2i_download.TestI2IDownload)` `AssertionError: Did not even create a session`

**Comment by lfdversluis** [comment]
retest this please

**Comment by lfdversluis** [comment]
Tribler.Test.test_i2i_download.TestI2IDownload.test_i2i_running (from nosetests)  `Did not even create a session`

**Comment by lfdversluis** [comment]
retest this please

**Comment by lfdversluis** [comment]
retest this please

**Comment by lfdversluis** [comment]
Tribler.Test.test_i2i_download.TestI2IDownload.test_i2i_running (from nosetests) Did not even create a session

**Comment by lfdversluis** [comment]
retest this please

**Comment by lfdversluis** [comment]
It's going to fail already..

**Comment by lfdversluis** [comment]
retest this please

**Comment by lfdversluis** [comment]
retest this please

**Comment by lfdversluis** [comment]
that i2i test works fine on my machine, even when running the whole batch like on jenkins...

**Comment by lfdversluis** [comment]
Passed, it's a miracle D:

**Comment by whirm** [comment]
There's two notes that havn't been adressed yet.

Also, this is not even touching code that's tested, why do you bother retesting? :D

**Comment by lfdversluis** [comment]
@whirm Ah sorry just hit end of page on my keyboard because of the retest this please spam.
RE retesting: I know, but its does give us info that this test is also a piece of junk.

**Comment by whirm** [comment]
squash it!

**Pull request merged**